pkg/registry/apis/provisioning/resources/client.go

```diff
@@ -15,11 +15,15 @@ import (
 	dashboardV2beta1 "github.com/grafana/grafana/apps/dashboard/pkg/apis/dashboard/v2beta1"
 	folders "github.com/grafana/grafana/apps/folder/pkg/apis/folder/v1beta1"
 	iam "github.com/grafana/grafana/apps/iam/pkg/apis/iam/v0alpha1"
+	"github.com/grafana/grafana/pkg/apimachinery/utils"
 	"github.com/grafana/grafana/pkg/services/apiserver"
 	"github.com/grafana/grafana/pkg/services/apiserver/client"
 )
 
 var (
+	FolderResourceInfo    = folders.FolderResourceInfo
```

Comment:
possible to use GVK or GVR?  the resourceInfo only exists for a few things, so will not be a pattern that could extend to any resource

Reply:
Its possible to go for GVK or GVR. ResourceInfo just seemed nice because it had GVK, GVR and SingularName under it.